static/src/javascripts/projects/commercial/modules/consentless/render-advert-label.spec.ts

```diff
@@ -62,14 +56,6 @@ describe('Rendering advert labels', () => {
 		});
 	});
 
-	it('Will not add a label if the adSlot already has one', async () => {
-		createAd(adverts['alreadyLabelled']);
```

Comment:
If we're deleting this we can also delete alreadyLabelled from the adverts record! Even more lines saved 😊